TCMSApp/src/client/app/layout/shell.controller.js

```diff
@@ -1,13 +1,13 @@
-(function() {
+(function () {
     'use strict';
 
     angular
         .module('app.layout')
         .controller('ShellController', ShellController);
 
-    ShellController.$inject = ['$rootScope', '$timeout', 'config', 'logger'];
+    ShellController.$inject = ['$rootScope', '$timeout', 'config', 'logger', '$state'];
     /* @ngInject */
-    function ShellController($rootScope, $timeout, config, logger) {
+    function ShellController($rootScope, $timeout, config, logger, $state) {
         var vm = this;
```

Comment:
we don't need this code here anymore, I removed it to navigation panel controller. 
shell is a place where all our project situated )